packages/client/src/game/entities/Entity.ts

```diff
@@ -0,0 +1,63 @@
+import { EntityTypes, GamePlayerType, HandEntityTypes } from '../types';
+import {
+  calcCanvasMaxSizes,
+  createCanvas,
+  createColorBox,
+  createLayer,
+  createNameLayer,
+  createUnoButton,
+} from '../utils';
+
+export class Entity<T extends EntityTypes> {
+  layer!: HTMLDivElement;
+  context!: CanvasRenderingContext2D;
+  name!: HTMLParagraphElement;
+  colorBox: HTMLDivElement | null = null;

```

Comment:
```suggestion
  private layer!: HTMLDivElement;
  private context!: CanvasRenderingContext2D;
  private name!: HTMLParagraphElement;
  private colorBox: HTMLDivElement | null = null;
```
Лучше явно указать что это приватные.